README.md

```diff
@@ -118,12 +118,14 @@ Potential statuses:
     ```bash
     php artisan key:generate
     ```
+1. Create a [GitHub Personal Access Token](https://github.com/settings/tokens) and update the  `GITHUB_TOKEN` key in your `.env` file.
```

Comment:
Not sure if it's worth noting here, but you have to use a "classic" PAT for GraphQL queries. I tested first with a fine-grained token, but looks like [the github cli hasn't been updated to support them yet](https://github.com/cli/cli/issues/6680). 🙃